package.json

```diff
@@ -2,27 +2,27 @@
   "name": "hyper-ts",
   "version": "0.7.6",
   "description": "Type safe middleware architecture for HTTP servers",
-  "files": [
-    "lib",
-    "es6"
-  ],
   "main": "lib/index.js",
   "module": "es6/index.js",
   "typings": "lib/index.d.ts",
```

Comment:
I guess these must be updated too, right?

Reply:
This might be the same as https://github.com/gcanti/fp-ts/pull/1447, where it's not possible to install outside of npm directly.

Reply:
If there is no way to test it, I can release an `alpha` version with this change and see if it actually works.

Reply:
I've released 0.7.7-0, I've tried and it seems ok to me.